### DISCUSSION TITLE
Unraid按照教程配置完后端后使用Jellyfin刮削报错，metatube-server日志没有报错，Jellyfin日志中有报错，unraid也设置了全局代理

### DISCUSSION THREAD

**Discussion by songyi-uh** [metatube-community/jellyfin-plugin-metatube#372] [Q&A]
Unraid按照教程配置完后端后使用Jellyfin刮削报错，metatube-server日志没有报错，Jellyfin日志中有报错，unraid也设置了全局代理，请问下有谁知道报错原因么？
![2%AH{4T3EX%DD}STC2L@%2V](https://github.com/user-attachments/assets/2da2fac8-79e1-4d1f-9f9a-0a35d314e2f7)

![Uploading 1.png…]()

**Comment by Topbcy** [comment]
Jellyfin docker 是用誰的?  哪一個版本

**Reply by songyi-uh** [reply]
用的是unraid市场上的官方Jellyfin，版本是10.9.8

**Reply by Topbcy** [reply]
你jellyfin裡面的metatube設定截圖我看一下~~

**Reply by songyi-uh** [reply]
![屏幕截图_28-7-2024_232934_192 168 100 226](https://github.com/user-attachments/assets/e4dfa8bf-898e-48fd-ade5-f1ca6fab8c45)
我用koyed搭建了一个后端，可以正常刮削不报错，其它的参数没有变动，把server参数改为用unraid运行的metatube后端地址就会报错

**Reply by Topbcy** [reply]
那可能還是那道牆的問題

**Reply by songyi-uh** [reply]
我把整个unraid的网络已经全局代理出去了，用部署在koyed上的后端正常刮削，用unraid运行的后端就是报错
